src/pages/user/Profile.js

```diff
@@ -0,0 +1,278 @@
+import React, { Component } from 'react';
+
+import { Link } from '@reach/router';
+
+import {
+  Container,
+  Row,
+  Col,
+  Card,
+  CardGroup,
+  ButtonGroup,
+  Button
+} from 'react-bootstrap';
+
+export default class Profile extends Component {
+  render() {
+    return (
+      <Container>
+        <Row className="text-center">
+          <Col>
+            <p>
+              This is what other users see when they look at your profile.
+              <br />
+              <Link to="/userSettings">Click here to edit your profile</Link>
+            </p>
+          </Col>
+        </Row>
+        <Row className="text-center">
+          <Col>
+            <h1>xXTeddyBearSlayer69Xx</h1>
+            <img src="/media/harold.jpeg" alt="Profile pic" />
+          </Col>
+        </Row>
+        <Row className="text-center">
+          <Col>
+            <ButtonGroup>
+              <Button variant="primary">
+                <span>Message</span>
+              </Button>
+              <Button variant="primary">
+                <span>Follow</span>
+              </Button>
+              <Button variant="primary">
+                <span>Report</span>
+              </Button>
+            </ButtonGroup>
+            <hr />
+          </Col>
+        </Row>
+        <Row className="text-center">
+          <Col>
+            <h2>Newest recipes</h2>
+            <CardGroup>
+              <Card>
+                <Card.Img
+                  variant="top"
+                  src="/media/card-test-1.jpg"
+                  className="img-fluid w-75 mx-auto"
+                />
+                <Card.Body>
+                  <Card.Title>
+                    <Link to="#">Strawberry Milkshake</Link>
```

Comment:
I'd prefer to have these templated out and filled with mock data rather than inlined but doesn't need to be done now as it'll likely take longer than we want to wait to merge this in.